page-objects/pages/managementPages/ManageClassPage.js

```diff
@@ -67,14 +69,34 @@ async function createNewClass({schoolYear, teachers, classGrade, className, cust
     if (customClassName) await setCustomClassName(customClassName);
     await clickAddClassConfirmation();
 }
-async function isClassEdited(newClassName, teacherLastname) {
-    const allClassesContainer = await waitHelpers.waitUntilElementIsVisible(classListTable)
-    const allClassesContent = await allClassesContainer.getText()
-    const contentArray = allClassesContent.split(" ")
+async function isClassEdited(newClassName, teacherLastname, expectedResult) {
+	if (!expectedResult) {
+		try {
+			const allClassesContainer = await waitHelpers.waitUntilElementIsVisible(classListTable)
+			const allClassesContent = await allClassesContainer.getText()
+			const contentArray = allClassesContent.split(" ")
+			expect(contentArray[0]).not.to.equal(newClassName)
+			expect(contentArray[1]).not.to.equal(teacherLastname)
+		} catch (e) {
```

Comment:
Sorry, I do not understand which error you are catching here? Is that in case no class is existing?

Reply:
yes, when there are 0 classes then allClassesContainer does not exist

Reply:
Okay, this makes sense. Thanks